src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star stars__star--1"></div>
+      <div class="stars__star stars__star--2"></div>
```

Comment:
No need in modifiers like `stars__star--1`

Reply:
Removed the redundant classes